packages/video-intelligence/package.json

```diff
@@ -1,27 +1,43 @@
 {
   "name": "@google-cloud/video-intelligence",
-  "repository": "googlecloudplatform/google-cloud-node",
   "version": "0.1.0",
   "author": "Google Inc",
   "description": "Google Cloud Video Intelligence API client for Node.js",
   "main": "src/index.js",
   "files": [
-    "src"
+    "src",
+    "AUTHORS",
```

Comment:
Without these additions, the license and related documents aren't published to npm.